services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/_meta.py

```diff
@@ -0,0 +1,33 @@
+""" Application's metadata
+
+"""
+from typing import Final
+
+from packaging.version import Version
+from servicelib.utils_meta import PackageInfo
+
+info: Final = PackageInfo(package_name="simcore-service-resource-usage-tracker")
+__version__: Final[str] = info.__version__
+
+
+PROJECT_NAME: Final[str] = info.project_name
+VERSION: Final[Version] = info.version
+API_VERSION: Final[str] = info.__version__
+API_VTAG: Final[str] = info.api_prefix_path_tag
+SUMMARY: Final[str] = info.get_summary()
+
+
+# NOTE: https://texteditor.com/ascii-frames/
+APP_STARTED_BANNER_MSG = r"""
+d8888b. d88888b .d8888.  .d88b.  db    db d8888b.  .o88b. d88888b        db    db .d8888.  .d8b.   d888b  d88888b        d888888b d8888b.  .d8b.   .o88b. db   dD d88888b d8888b.
```

Comment:
Q: you did not like any of my shorter name versions? 

FACT: You have a terrible taste for banners :-)))))

Reply:
I found them not very metallic... sorry :rofl: